debian/control

```diff
@@ -115,6 +116,7 @@ Depends: ${misc:Depends}, vlan, make,
  python-twisted-bin (<=13.0), python-twisted-core (<=13.0),
 # EAP TLS
  libcrypt-smime-perl
+ redis
```

Comment:
Duplicate (also don't forget to add a comma on the line before when you add a new dep)